Sources/SwiftBlade/Utils.swift

```diff
@@ -29,14 +32,17 @@ func getRemoteConfig(network: HederaNetwork, dAppCode: String, sdkVersion: Strin
     }
 }
 
-func getVisitorId(fingerPrintApiKey: String) async throws -> String {
+func getVisitorId(_ remoteConfig: RemoteConfig) async throws -> String {
     do {
-        let customDomain: Region = .custom(domain: "https://identity.bladewallet.io")
-        let configuration = Configuration(apiKey: fingerPrintApiKey, region: customDomain)
+        let customDomain: Region = .custom(domain: remoteConfig.fpSubdomain)
+        let configuration = Configuration(apiKey: remoteConfig.fpApiKey, region: customDomain)
         let client = FingerprintProFactory.getInstance(configuration)
         let visitorId = try await client.getVisitorId()
         return visitorId
     } catch {
-        throw error
+        let configuration = Configuration(apiKey: remoteConfig.fpApiKey)
```

Comment:
It's a Fallback for subdomain error, right?

Reply:
correct